src/schema/schema.rs

```diff
@@ -358,6 +383,21 @@ impl Schema {
         }
         Ok(doc)
     }
+
+    /// Searches for a full_path, returning the field name and a json path.
+    pub fn find_field<'a>(&self, full_path: &'a str) -> Option<(Field, &'a str)> {
+        if let Some(field) = self.0.fields_map.get(full_path) {
```

Comment:
No unit tests!?

Reply:
I mostly copied the same unit test on `QueryParser`. I can remove the test from `QueryParser` or add more tests to `Schema` if that's what we want.

Reply:
yes please that would be awesome!